examples/error-user-defined/error_user_defined.description

```diff
@@ -0,0 +1,7 @@
+// We can define your own error types to match the error we are trying to model.
+// User defined error types in Ballerina has to have a reason value which is a subtype of string,
+// and a detail type which is a subtype of `record {| string message?; error cause?; (anydata|error)...; |}`.
+// The default error detail record is define in lang.error module of the lang library as `Detail`.
```

Comment:
Do we need this?

Reply:
Wrote is for the completeness, will remove it